src/Sets/Zonotope.jl

```diff
@@ -275,7 +275,7 @@ A new zonotope with less generators, if possible.
 
 See `overapproximate(Z::Zonotope{N}, ::Type{<:Zonotope}, r::Union{Integer, Rational}) where {N<:Real}` for details.
 """
-function reduce_order(Z::Zonotope{N}, r::Union{Integer, Rational})::Zonotope{N} where {N<:Real}
+function reduce_order(Z::Zonotope{N}, r::Union{Integer, Rational}) where {N<:Real}
```

Comment:
I don't know if we want to keep the type annotation in the docstring; i'm fine either way..

Reply:
Same, but if in doubt, let us remove it.